opm/parser/eclipse/RawDeck/StarToken.cpp

```diff
@@ -69,55 +71,38 @@ namespace Opm {
 
     template<>
     int readValueToken< int >( string_view view ) {
+        int n = 0;
```

Comment:
I really don't understand this code - to me it seems the code should return `n= 0`; or raise an exception. I would have expected to pass `&n`?